src/cli/miner_cli_genesis.erl

```diff
@@ -40,21 +42,25 @@ register_all_cmds() ->
                    genesis_load_cmd(),
                    genesis_export_cmd(),
                    genesis_key_cmd(),
-                   genesis_proof_cmd()
+                   genesis_proof_cmd(),
+                   genesis_export_ledger_cmd(),
+                   genesis_recreate_ledger_cmd()
                   ]).
 %%
 %% genesis
 %%
-
 genesis_usage() ->
     [["genesis"],
      ["miner genesis commands\n\n",
-      "  genesis create <old_genesis_file> <pubkey> <proof> <addrs>  - Create genesis block keeping old ledger transactions.\n",
-      "  genesis forge <pubkey> <key_proof> <addrs>                  - Create genesis block from scratch just with the addresses.\n",
-      "  genesis load <genesis_file>                                 - Load genesis block from file.\n"
-      "  genesis export <path>                                       - Write genesis block to a file.\n"
-      "  genesis key                                                 - create a keypair for use as a master key\n"
-      "  genesis proof <privkey>                                     - create a key proof for adding a master key to the genesis block\n"
+      "  genesis create <old_genesis_file> <pubkey> <proof> <addrs>         - Create genesis block keeping old ledger transactions.\n"
+      "  genesis forge <pubkey> <key_proof> <addrs>                         - Create genesis block from scratch just with the addresses.\n"
+      "  genesis load <genesis_file>                                        - Load genesis block from file.\n"
+      "  genesis export <path>                                              - Write genesis block to a file.\n"
+      "  genesis key                                                        - create a keypair for use as a master key\n"
+      "  genesis proof <privkey>                                            - create a key proof for adding a master key to the genesis block\n"
+      "  genesis export_ledger <masterkey> <txn_list_output_path>           - export data from ledger as a list of txns including accounts, gateways, validators and chainvars\n"
+      "  genesis recreate_ledger <n> <curve> <path_to_txn_list> <addrs>     - export data from ledger as a list of txns including accounts, gateways, validators and chainvars\n"
```

Comment:
This has copied help text from export_ledger

Reply:
fixed here: https://github.com/helium/miner/pull/1467/commits/1675824bf67a04674899cb02b6cb681c7afe8601